services/spar/src/Spar/Scim/User.hs

```diff
@@ -336,12 +360,13 @@ createValidScimUser ScimTokenInfo {stiTeam} vsu@(ST.ValidScimUser veid handl mbN
       veid
 
   -- If applicable, trigger email validation procedure on brig.
-  lift $ validateEmailIfExists buid veid
+  lift $ ST.runValidExternalId (validateEmailIfExists buid) (\_ -> pure ()) veid
 
   -- {suspension via scim: if we don't reach the following line, the user will be active.}
   lift $ do
     old <- Brig.getStatus buid
-    let new = ST.scimActiveFlagToAccountStatus old (Just active)
+    let new = ST.scimActiveFlagToAccountStatus old active
+        active = Scim.active . Scim.value . Scim.thing $ storedUser
```

Comment:
explanation for the reviewer: `active` used to come from a function argument, but in the flow implemented here, brig will change that value, and we need to listen to brig, not the scim peer who is creating the user.